mora/service/address.py

```diff
@@ -256,3 +262,78 @@ def edit_address(employee_uuid, req):
     }
 
     c.bruger.update(payload, employee_uuid)
+
+
+@blueprint.route('/address_lookup/')
```

Comment:
I think we should move this to `/o/<orgid>/` — both for consistency and clarity.

Reply:
Having the endpoint be located at `/o/<orgid>/` might make the user believe that the address search is automatically local to that specific organisation, which it isn't.

Reply:
The `local` argument actually works that way — mostly, anyway — and the UI actually does work that way.

Reply:
My point is, it would then be possible to search for an address on `/o/<org 1>/address_lookup?q=asd&orgid=<org 2>`, which would search for addresses in org 2, not org 1.

Reply:
Sorry, perhaps I wasn't clear; what I suggested was:

    /o/<org 1>/address_lookup?q=asd

_Instead_ of the `orgid`/`local` parameter. Then, in order to search for adresses in the entire country, you'd do:

    /o/<org 1>/address_lookup?q=asd&global=1

Or something similar…

Reply:
Right, that makes sense. I'll change it.

Reply:
Great 🙂 I think also forgot to mention this earlier, but I'd suggest you include ‘autocomplete’ in the URL endpoint — it's what we use it for.